### ISSUE TITLE
Yarn dev:inspect returns unknown or unexpected option

### ISSUE THREAD

**Issue by Tjitse-E** [DivanteLtd/vue-storefront-api#271]
I'm trying to work on the VS API and i've noticed that the `--inspect` flag doesn't work together with Nodemon after the latest typescript changes.

When running `yarn dev:inspect`:

```
yarn run v1.15.2
$ nodemon -w src --exec "ts-node --inspect src"
[nodemon] 1.19.1
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: /Users/tjitse/sites/vue-storefront-api/src/**/*
[nodemon] starting `ts-node --inspect src`
/Users/tjitse/sites/vue-storefront-api/node_modules/arg/index.js:92
						throw err;
						^

Error: Unknown or unexpected option: --inspect
    at arg (/Users/tjitse/sites/vue-storefront-api/node_modules/arg/index.js:90:19)
    at Object.<anonymous> (/Users/tjitse/sites/vue-storefront-api/node_modules/ts-node/dist/bin.js:14:12)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:829:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
[nodemon] app crashed - waiting for file changes before starting...
```

I've investigated this further, but I wan't able to get the `--inspect` flag working with Nodemon in combination with Typescript. 

I have a possible solution for this, which I will [propose in a new PR](https://github.com/DivanteLtd/vue-storefront-api/pull/272).

### Tested with
- Node v8.x and v10.x
- VS API develop branch. [Latest commit](https://github.com/DivanteLtd/vue-storefront-api/commit/5f78af2d85f314bcbc168d85e5c514b8ee4a55b7)
